packages/terra-core-docs/src/terra-dev-site/doc/table/example/DefaultTable.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Table from 'terra-table';
+
+const tableData = {
+  cols: [
+    { id: 'Column-0', displayName: 'Patient' },
+    { id: 'Column-1', displayName: 'Location' },
+    { id: 'Column-2', displayName: 'Illness Severity' },
+    { id: 'Column-3', displayName: 'Visit' },
+    { id: 'Column-4', displayName: 'Allergy' },
+    { id: 'Column-5', displayName: 'Primary Contact' },
+    { id: 'Column-6', displayName: 'Generic Order Counts' },
+    { id: 'Column-7', displayName: 'Patient Age' },
+    { id: 'Column-8', displayName: 'Medication History' },
+    { id: 'Column-9', displayName: 'My Relationship' },
+    { id: 'Column-10', displayName: 'Not Selectable' },
+  ],
+  rows: [
+    {
+      id: '1',
+      cells: [
+        { content: 'Fleck, Arthur' },
+        { content: '1007-MTN' },
+        { content: 'Unstable' },
+        { content: 'Inpatient, 2 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+        { isMasked: true },
+        { isMasked: true },
+        { content: 'Admitting Physician' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '2',
+      cells: [
+        { content: 'Wayne, Bruce' },
+        { content: '1007-MTN-DR' },
+        { content: 'Stable' },
+        { content: 'Outpatient, 2 days' },
+        { content: 'Phytochemicals' },
+        { content: 'Grayson, Richard' },
+        { content: '' },
+        { content: '' },
+        { isMasked: true },
+        { content: 'Admitting Physician' },
+        { content: '' },
+      ],
+    },
+  ],
+};
+
+const DefaultTable = () => (
+  // eslint-disable-next-line react/forbid-dom-props, jsx-a11y/no-noninteractive-tabindex
+  <div style={{ overflow: 'scroll' }} tabIndex={0}>
```

Comment:
There should not be a scroll component added around the Terra Table.  The Table component should have its own scrollbar.

Reply:
@cm9361 I advised this change after a sync with Kenny since: 
- we won't need to manually set scroll position like we do with grid.
- because we're not managing scroll position, any scrolling container will need to be focusable for keyboard control when there's no focusable elements in the table. So rather than having an always focusable scroll area, we can leave that to the discretion of the consumer.

If this change is technically incompatible with sticky headers or pinned columns or there's some other reason to have it in the component, we should discuss.

Reply:
I think the scrollbar should live within the Terra Table element.  We don't want to rely on consumers providing a scroll container so that functionality such as pinned columns or being able to see data beyond the bounds of the container works.  It also allows us to properly control the tabIndex on the scroll container.  If we leave this up to consumers, we leave ourselves open to issues.

Reply:
Also, this component will be used by the grids as well.

Reply:
Alrighty. I'm cool with it. There should be a prop to disable it for Table though. We won't always want the scroll container.